### ISSUE TITLE
RUN

### ISSUE THREAD

**Issue by janriecarlostidres** [Noctem/pokeminer#328]
Can I run de monocle in eclipse or can i run only cmd(pronpt) of windows, or double click?

**Comment by M3G4THEKING** [comment]
@janriecarlostidres  yes you can .and search "How to add Python3.6 as interpreter in Eclipse"
